mix.exs

```diff
@@ -53,13 +54,14 @@ defmodule Bamboo.Mixfile do
   defp deps do
     [
       {:plug, "~> 1.0"},
-      {:ex_machina, "~> 2.2", only: :test},
+      {:mime, "~> 1.4"},
+      {:ex_machina, "~> 2.4", only: :test},
       {:cowboy, "~> 1.0", only: [:test, :dev]},
-      {:phoenix, "~> 1.1", only: :test},
+      {:phoenix, "~> 1.1", optional: true},
       {:phoenix_html, "~> 2.2", only: :test},
-      {:excoveralls, "~> 0.4", only: :test},
-      {:floki, "~> 0.8", only: :test},
-      {:ex_doc, "~> 0.20", only: :dev},
+      {:excoveralls, "~> 0.13", only: :test},
+      {:floki, "~> 0.29", only: :test},
+      {:ex_doc, "~> 0.23", only: :dev},
```

Comment:
Could you provide information as to why we needed these changes? Did we have to update dependencies?

Reply:
Force of habit, since all of these deps are limited to :dev and :test there should not be any issues with updating.

Reply:
We add `:mime` since it is used in `lib/bamboo/attachment.ex` and not directly referenced.

Reply:
We mark phoenix as optional since it is used in `lib/bamboo/phoenix.ex` with `Phoenix.View.render_to_string`. It's not 100% necessary to use that module but it's very likely to be used.